### PR TITLE
fix(Row): update TS structure

### PR DESCRIPTION
**Pull request by jazzyw**
## Description

Updates Row's TS declaration file to match the new TS structure ([@lightningjs/core subclassable components guide](https://github.com/rdkcentral/Lightning/blob/2e4c829be45614290405c222bf102022a6219153/docs/TypeScript/Components/SubclassableComponents.md))

## References

LUI-841

## Testing

- Create a new component using `yarn createComponent @lightningjs/ui-components MyComponent`
- Make sure the component extends `Row` (rather than Base)
- Ensure the `Row` properties/methods show up when using the dot syntax or through patch

## Checklist

- [x] all commented code has been removed
- [x] any new console issues have been resolved
- [x] code linter and formatter has been run
- [x] test coverage meets repo requirements
- [x] PR name matches the expected semantic-commit syntax

### CODE REVIEW

**Review by arwehrman — APPROVED** [review]

**Review by anthony9187 — APPROVED** [review]

**Comment by jazzyw** [comment]
Waiting for #231 to be merged in before merging this one.

**Pull request merged**
